cli/tests/integration_tests.rs

```diff
@@ -2764,6 +2764,13 @@ mod util {
       return false;
     }
 
+    // If the first line of the pattern is just a wildcard the newline character
+    // needs to be pre-pended so it can safely match anything or nothing and
+    // continue matching.
+    if pattern.lines().next() == Some(wildcard) {
+      s.insert_str(0, "\n");
+    }
+
```

Comment:
Can this bit be exercised in test (`test_pattern_match`)?

Reply:
I'll get on that

Reply:
Tests added